app/src/main/java/org/oppia/app/drawer/NavigationDrawerItem.kt

```diff
@@ -0,0 +1,19 @@
+package org.oppia.app.drawer
+
+import org.oppia.app.R
+
+enum class NavigationDrawerItem constructor(private val value: Int) {
```

Comment:
This can be changed to `enum class NavigationDrawerItem {`

Reply:
This enum need to have the value. Updated to NavigationDrawerItem(val value: Int)